spec/services/proofing/aamva/proofer_spec.rb

```diff
@@ -612,35 +643,12 @@ def self.test_not_successful
             sex: 1,
             weight: 1,
             eye_color: 1,
+            dob: 1,
           },
         )
       end
     end
 
-    context 'when issue / expiration present' do
```

Comment:
Just out of ignorance on my part, why isn't this test needed anymore?

Reply:
This bit of functionality is covered in the VerificationRequest spec and the individual attribute specs further up.